src/main/resources/META-INF/plugin.xml

```diff
@@ -17,6 +17,7 @@
                             className="com.virtuslab.gitmachete.frontend.ui.impl.root.GitMacheteContentProvider"
                             predicateClassName="com.virtuslab.gitmachete.frontend.ui.impl.root.GitMacheteVisibilityPredicate"/>
         <errorHandler implementation="com.virtuslab.gitmachete.frontend.ui.impl.GitMacheteErrorReportSubmitter"/>
+        <applicationService serviceImplementation="com.virtuslab.gitmachete.backend.services.GitMacheteRepositoryCacheService"/>
```

Comment:
application service - so it is global to all projects. pls confirm that everything works good with 2 (or more) projects openeed

Reply:
Tested it manually and I get cache hits when switching projects and opening the Git Machete Tab.

Reply:
cool